imports/ui/pages/root-redirector.js

```diff
@@ -8,6 +8,17 @@ import './root-redirector.html';
 Template.app_rootRedirector.onCreated(() => {
   if (Meteor.userId()) {
     Template.instance().subscribe('memberships.ofUser', { userId: Meteor.userId() });
+    Template.instance().autorun(() => {
```

Comment:
Ugy latom ez a main_layout-bol jott.
Mivel ez mar azert egy komolyabb 10 soros kod blokk, javaslom inkabb tegyuk ki egy kulon fuggvenybe, es mindket helyrol azt a fuggvenyt hivjuk.
Ekkora kod blokkokat mar nem celszeru atmasolni. Addig persze szabad amig megnezzuk hogy mukodik e igy. De aztan a vegen talalnunk kell neki egy jo helyet.
Fv nevnek javaslom a setActiveCommunity()-t, es hogy hova... hat vagy  communities.js-be paszirozzuk be egy if (Meteor.isClient)-ben (hiszen session van benne) vagy csinaljunk egy uj fajlt active-community.js nevvel, ami csak kliens oldalon lesz importalva, ezekben a layoutokban. Utobbi talan meg tisztabb.

Reply:
Igazbol miert van szuksege az appRootRedirector-nak, hogy feliratkozzon a user membershipjeire, es az active community-re?
Nem csak annyi a dolga hogy megnezze van e belepett user, ha nincs akkor -> intro page, ha van akkor -> Board
aztan onnan a main layout mar intezi a tobbit.
Szoval lehet nem is kell ez a kod ide.

Reply:
Az a rész pont nincs itt a kódból: a board és a communities list között választ az alapján, hogy van-e active community. Ki lehet venni innen, de akkor a boardra kell egy átirányítás, hogy ha nincs active communityje, akkor menjen tovább a communities listre.

Reply:
Ja, OK, az mas. Akkor kell ide.
Viszont azzal rendszresen tapasztalok olyat hogy ugyan van hazam, megis a hazvalasztora dob. (masteren is) Szoval per pill az nem tokeletesen mukodik. ilyen idozitesi problemea lehet ott is, ha valami elobb jon meg, akkor nem veszi szre hogy van hazam, aztan mire eldobott mar mar hiaba jon meg, mert el lettem iranyitva, jellegu...
Lehet hogy le kene egyszerusiteni valahogy ezt az egeszet?
Te szereted az egyszerusiteseket. :)

Reply:
A jovoben akartuk, hogy az activeCommunity legyen elmentve, hogy ha tobb van neki, akkor ne mindig az elsore dobjuk, ha elotte legutobb a masodikban tevekenykedett.
Lehet hogy ez az uj feature, ezt is jelentosen leegyszerusitene?
Ehhez a Session-bol at kene rakni az activeCommunity mezot, a user-re.
(Eleg sok helyen kellene atirni, de eleg straightforward)
Az utan a user-nek eleg lenne sajat user objektumat latni, ahhoz hogy korrektul redirektalni lehessen.

Reply:
Eloszor persze gondoljuk at ennek az implikacioit, mielott ugy dontunk hogy atirjuk.

Reply:
Addig is refaktoráltam.